developer_manual/app/advanced/code_signing.rst

```diff
@@ -48,22 +48,19 @@ Is Code Signing Mandatory For Apps?
 Code signing is optional for all third-party applications. Applications 
 with a tag of "Official" on https://marketplace.owncloud.com/ require code signing.
 
-Technical details
+Technical Details
 -----------------
 
-ownCloud uses a X.509 based approach to handle authentication of code. Each 
-ownCloud release contains the certificate of a shipped ownCloud Code Signing 
-Root Authority. The private key of this certificate is only accessible to the 
-project leader, who may grant trusted project members with a copy of this 
-private key.
+ownCloud uses a X.509 based approach to handle authentication of code. 
+Each ownCloud release contains the certificate of a shipped ownCloud Code Signing 
+Root Authority. 
+The private key of this certificate is only accessible to the project leader, who may grant trusted project members with a copy of this private key.
```

Comment:
I think this handling changed. You can get infos from @felixboehm or @DeepDiver1975